packages/react/src/BaseStyles.tsx

```diff
@@ -52,77 +19,29 @@ export type BaseStylesProps = PropsWithChildren & {
   SystemCommonProps &
   SxProp
 
-function BaseStyles(props: BaseStylesProps) {
-  const {children, color, fontFamily, lineHeight, className, as: Component = 'div', style, ...rest} = props
+function BaseStyles({
+  children,
+  color,
+  fontFamily,
+  lineHeight,
+  className,
+  as: Component = 'div',
+  style,
+  ...rest
+}: BaseStylesProps) {
   const {colorMode, colorScheme, dayScheme, nightScheme} = useTheme()
-  const enabled = useFeatureFlag(CSS_MODULES_FEATURE_FLAG)
-
-  if (enabled) {
-    const newClassName = clsx(classes.BaseStyles, className)
-    const baseStyles = {
-      ['--BaseStyles-fgColor']: color,
-      ['--BaseStyles-fontFamily']: fontFamily,
-      ['--BaseStyles-lineHeight']: lineHeight,
-    }
-
-    // If props includes TYPOGRAPHY or COMMON props, pass them to the Box component
-    if (includesSystemProps(props)) {
-      const systemProps = getTypographyAndCommonProps(props)
```

Comment:
Can you clarify why this was needed before but not anymore? I'm confused since this seems to be the old-enabled version 😅

Reply:
This was added to check if system props were coming in, but we've abstracted that same functionality to the `BoxWithFallback` component so no need to double check now https://github.com/primer/react/blob/36f100706a32d369d1f464caeaa60ff38b576d30/packages/react/src/internal/components/BoxWithFallback.tsx#L11